src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/AssessmentLifecycleService.kt

```diff
@@ -21,8 +22,8 @@ class AssessmentLifecycleService {
   }
 
   fun eligibilityAnswerSubmitted(assessment: AssessmentWithEligibilityProgress): AssessmentStatus {
-    return when {
-      assessment.isChecksPassed() ->
```

Comment:
this was more complicated than it needed to be.. I've removed these functions and provided a single calculateStatus flag